src/gui/editors/PianoRoll.cpp

```diff
@@ -805,7 +805,7 @@ void PianoRoll::setBackgroundShade( const QColor & c )
 
 
 
-void PianoRoll::drawNoteRect( QPainter & p, int x, int y, 
+void PianoRoll::drawNoteRect( QPainter & p, int x, int y,
 				int width, const Note * n, const QColor & noteCol,
```

Comment:
@zonkmachine agreed.  Harder to cherry-pick too as the more lines that get modified increases the chances of a merge conflict.